statsmodels/stats/power.py

```diff
@@ -223,6 +223,9 @@ def solve_power(self, **kwds):
             import warnings
             from statsmodels.tools.sm_exceptions import HypothesisTestWarning
             warnings.warn('Warning: Effect size of 0 detected', HypothesisTestWarning)
+            raise NotImplementedError("`power` and `alpha` variables are "
+                                      "not defined.  This section of "
+                                      "statsmodels is not maintained.")
```

Comment:
AFAICS or AFAIR, the NotImplementedError is just wrong her.

The code should work as expected. If effect size is zero, then null and alternative hypothesis are the same and power == alpha.